### PR TITLE
Update libraries index.md (zh_cn)

### PR DESCRIPTION
**Pull request by alex-marmot**
I have updated the content according to the zh_tw form.

Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
Could you update your pull request body into English, so other people could know the purpose of this pull request, thanks!

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia

Please review this pull request. :pray:

File: zh_cn/libraries/index.md [hunk 1]
```diff
@@ -1,62 +1,99 @@
...
 
-{% include out-of-date.html %}
+与大多数的编程语言一样，Ruby也受益于海量的第三方类库。
+{: .summary}
 
-在别的地方还有不计其数的有趣并且有用的 Ruby 代码库，在它们中很多是以便捷的 *gem*
-文件格式发布的，其它的则是通过将源代码目录打包成归档文件（例如 .zip 或
-.tar.gz）进行发布。现在让我们一起瞧一瞧怎么寻找并安装代码库，来让它们为你所用。
+这些类库大部分都以 **Gem** 形式发布。 [**RubyGems**][1] RubyGems 是设计用来帮助创建，分享和安装
+这些类库的（从某种意义上来讲，RubyGems是一个类似于apt-get的分布式包管理系统，只不过专注于Ruby软件）。从Ruby1.9起，Ruby默认自带RubyGems，而之前的版本Ruby版本需要先[手动安装][2]。
```
Comment (hlcfan) [inline zh_cn/libraries/index.md]: **Ruby 1.9**

File: zh_cn/libraries/index.md [hunk 2]
```diff
@@ -1,62 +1,99 @@
...
 
+### 查找类库
 
-#### 搜索 Gem
+[**RubyGems.org**][1] 站点是类库的主要集散地，它提供gem形式的类库。你可以直接访问其网站或者使用 `gem` 命令。
 
-**搜索**命令可以通过确定的名字来搜索 gem。要搜索一个名字里包含了 html 关键词的 gem：
+使用 `gem search -r`，能查看 RubyGems 的 Repository。比如要找 rails，輸入 `gem search -r rails` 会返回与 Rails 相关的 Gem。使用`--local` (`-l`) 选项就会在本地你已安装的gem中搜索与Rails相关的Gem。使用`gem install [gem]`来安装 Gem 。可通过`gem list`来浏览已安装的Gem。更多关于 `gem` 命令的说明，请继续阅读，或前往 [RubyGems 的文件][3]。
+
+除了 RubyGems.org 之外，也有其它的站点可供查找类库。[RubyForge][4] 曾是 Ruby 类库之家，但近年來 [**GitHub**][5] 作为主要的Ruby相关资源的Repository而崛起。現在，通常 Gem 的源代码会被放在 GitHub，而 Gem 会被发布到 RubyGems.org。
```
Comment (hlcfan) [inline zh_cn/libraries/index.md]: **资源的 Repository 而崛起**

File: zh_cn/libraries/index.md [hunk 3]
```diff
@@ -1,62 +1,99 @@
...
-$ gem search html --remote
+$ gem search -r html
 
 *** REMOTE GEMS ***
 
 html-sample (1.0, 1.1)
 {% endhighlight %}
 
-(*参数标志`--remote` / `-r` 表明我们将要搜索的是 RubyGems.org 的官方 Gem。*)
+`--remote`或`-r` 标识表示我们想要查看RubyGems.org的官方仓库（默认设置）。使用`--local`或`-l`标识表示你想要搜索本地你已安装的gem包。
```
Comment (hlcfan) [inline zh_cn/libraries/index.md]: **查看 RubyGems.org 的官方仓库**

File: zh_cn/libraries/index.md [hunk 4]
```diff
@@ -1,62 +1,99 @@
...
 
-#### 安装一个 Gem
+#### 安裝 Gem
 
-当你知道你想**安装**哪个 Gem 的时候:
+一旦你知道你所需要安装的gem包，比如说颇受欢迎的rails:
```
Comment (hlcfan) [inline zh_cn/libraries/index.md]: **欢迎的 rails**

**Comment by alex-marmot** [comment]
I have format the content according to the https://github.com/sparanoid/chinese-copywriting-guidelines#spacing-c4

**Comment by JuanitoFatas** [comment]
Thanks for your contribution @alex-marmot, I changed “类库” back to “代码库” because the rubygem not only provide classes, could also be other things. Thanks for review @hlcfan.

**Comment by alex-marmot** [comment]
Oh. I see.

2015-05-30 13:32 GMT+08:00 Juanito Fatas <notifications@github.com>:

**Pull request closed without merging**